doc/pages/administrator/095-advanced-operations.md

```diff
@@ -134,3 +134,124 @@ After you end performing all the steps above, the interface will be completely r
 You can then proceed to install a new interface with the same name and major version without any
 conflict. *Remember to remove the interface also on the device side, otherwise devices will keep
 getting disconnected if they try to publish on the deleted interface.*
+
+## Backup your Astarte resources
+
+Backing up your Astarte resources is crucial in all those cases in which your Astarte instance have
+to be restored after an unforeseen event (e.g. accidental deletion of resources, deletion of the
+Operator - as it will be discussed later on - etc.).
+
+A full recovery of your Astarte instance along with all the persisted data is possible **if and only
+if** your Cassandra StatefulSet is deployed independently from Astarte, i.e. it must be deployed
```

Comment:
call it "Cassandra/Scylla instance" instead of StatefulSet, since usually if it's external it is not even a StatefulSet